azure-iot-device/azure/iot/device/provisioning/pipeline/pipeline_stages_provisioning.py

```diff
@@ -44,3 +58,388 @@ def _run_op(self, op):
 
         else:
             super(UseSecurityClientStage, self)._run_op(op)
+
+
+class CommonProvisioningStage(PipelineStage):
+    @pipeline_thread.runs_on_pipeline_thread
+    def _get_error(self, provisioning_op, prov_op_name, error):
```

Comment:
>prov_op_name [](start = 42, length = 12)

Instead of passing prov_op_name as a parameter, why not use provisioning_op.request_type?